lib/req/request.ex

```diff
@@ -524,7 +524,7 @@ defmodule Req.Request do
       iex> Req.Request.get_option_lazy(req, :b, fun)
       42
   """
-  @spec get_option_lazy(t(), atom(), (-> term())) :: term()
+  @spec get_option_lazy(t(), atom(), (() -> term())) :: term()
```

Comment:
got this from running mix format

Reply:
this is because you're probably using older Elixir version. We always use the latest for formatting!